### DISCUSSION TITLE
How do I db-dump from a specific branch? I have branching enabled

### DISCUSSION THREAD

**Discussion by vineethvijayan314** [supabase/supabase#31153] [Questions] [answered]
I have been trying to do a db dump from a specific branch, I tried linking but it only downloads from the main db, I tried db-url but it shows an error Connection refused Is the server running on that host and accepting TCP/IP connections?

I also checked is there is any network bans, but there is not.

**Comment by vineethvijayan314** [comment]
I tried the same with main branch which is working fine

**Reply by sweatybridge** [reply, marked as answer]
Have your tried using the pooler url (session mode) of the branch? ie.

```
supabase db dump --db-url 'postgresql://postgres.branch-project-ref:[YOUR-PASSWORD]@aws-0-ap-southeast-1.pooler.supabase.co:5432/postgres'
```
